src/config/index.ts

```diff
@@ -14,6 +14,9 @@ export const TESTING_ENV = process.env.ENV === 'test';
 export const CI_ENV = process.env.ENV === 'ci';
 export const JWT_SECRET_DEFAULT = 'some-secret-string-default';
 
+export const USE_AWS_SES = () => process.env.USE_AWS_SES === 'true';
```

Comment:
Now I see why you call a function. But, is there a way to call the function right away and just use the USE_AWS_SES env prop as a value? Just to keep consistency among the other env variables. 
I think something like this should work:

```
export const USE_AWS_SES =   process.env.USE_AWS_SES === 'true' ? true: false
```

Reply:
(2) If `USE_AWS_SES` happens to be a `bool` value, then there's no need for this, you should always use a bool directly. There's even a `.ts` rule to check this : 

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unnecessary-boolean-literal-compare.md

So instead of creating an arrow function, you could just pass `USE_AWS_SES` in the first place

Reply:
fixed ! We decide to use one environment variable with the name of the transporter to use ( for example EMAIL_TRANSPORTER=AWS ), instead the true/false options.
Thanks!